src/client/ui/filter_stories.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+export default class FilterText extends React.Component {
```

Comment:
Always use a noun for the Class name. `TextFilter` would be a better name.
And make sure file name matches that too. (`text_filter.js`)